etc/config/solidity.amazon.properties

```diff
@@ -24,14 +24,18 @@ compiler.solc0821.exe=/opt/compiler-explorer/solc-0.8.21/solc
 compiler.solc0821.semver=0.8.21
 compiler.solc0821.name=solc 0.8.21
 
-group.zksolc.compilers=zksolc140
+group.zksolc.compilers=zksolc141:zksolc150
 group.zksolc.compilerType=solidity-eravm
 group.zksolc.supportsBinary=false
 group.zksolc.instructionSet=eravm
-compiler.zksolc140.exe=/opt/compiler-explorer/zksolc-1.4.0/zksolc
```

Comment:
By removing this ID you will cause anyone who has bookmarked or made short links to the 140 to have broken links (it will pick the default compiler instead). If you want to fix this, send a follow up PR that adds an `.alias` to the `zkcolv141` compiler (grep about to find other examples)